packages/shared/lib/sdk/sync.ts

```diff
@@ -13,8 +13,9 @@ import { Nango } from '@nangohq/node';
 import configService from '../services/config.service.js';
 import paginateService from '../services/paginate.service.js';
 import proxyService from '../services/proxy.service.js';
-
-type LogLevel = 'info' | 'debug' | 'error' | 'warn' | 'http' | 'verbose' | 'silly';
+import axios from 'axios';
+import { getPersistAPIUrl } from '../utils/utils.js';
+import type { LogLevel } from '../models/Activity.js';
```

Comment:
@TBonnin FYI any type needs to not be imported which is why all the types are in this file. This causes this error as reported in [Slack](https://nangohq.slack.com/archives/C06GHQPLTAN/p1706859608331169)